src/main/java/org/jabref/gui/entryeditor/FieldsEditorTab.java

```diff
@@ -97,7 +97,7 @@ private void setupPanel(BibEntry entry, boolean compressed) {
             fieldEditor.bindToEntry(entry);
 
             editors.put(field, fieldEditor);
-            labels.add(new FieldNameLabel(field));
+            labels.add(new FieldNameLabel(field, field.getName()));
```

Comment:
Why do you use field.getName() as an argument even though you already set field as the first argument? Seems like an unnecessary duplication.

Reply:
You're right. I have changed it. Thank you